components/modal/examples/10-sizing.js

```diff
@@ -0,0 +1,90 @@
+import React, { useReducer } from 'react';
```

Comment:
Can we use 'size' as the example's menu item... as per buttons etc

`examples/10-size.js`